x-pack/plugins/integration_assistant/server/graphs/categorization/prompts.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ChatPromptTemplate } from '@langchain/core/prompts';
+
+export const CATEGORIZATION_MAIN_PROMPT = ChatPromptTemplate.fromMessages([
```

Comment:
Should we eventually store them in a database and provide versions? Just to have it similar to the python app?

Reply:
Don't think that's needed no, as we will have versions now through the github commits, and we don't really want to introduce any more dependencies to the Kibana application itself.

Looking at the other GenAI plugins they also store their prompts locally.
We then also get them as part of PR's which is nice :)